src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -74,11 +78,46 @@
   static final String ENCRYPTED_FILE_EXTENSION = ".enc";
   static final String MISSING_FILE_EXTENSION = ".missing";
 
+  static final File cacheFile = new File(getBriefcaseFolder(), "cache.ser");
+
+  static Map pathToHashMap;
```

Comment:
Small, nit-picking preference to calling this something other than ___HashMap (just to avoid any potential confusion).  pathToMd5Map perhaps?